beacon-chain/core/transition/skip_slot_cache_test.go

```diff
@@ -80,6 +82,8 @@ func TestSkipSlotCache_ConcurrentMixup(t *testing.T) {
 		require.NoError(t, err)
 		s1, err = transition.ExecuteStateTransition(context.Background(), originalState.Copy(), wsb)
 		require.NoError(t, err, "Could not run state transition")
+		s1, err = transition.ProcessSlot(context.Background(), s1)
```

Comment:
This current test setup is unrealistic in the production environment. The state root is zeros until we call `ProcessSlot`, this only applies to genesis condition